src/zns.ts

```diff
@@ -3,23 +3,44 @@ import { Contract } from '@zilliqa-js/contract';
 import { toChecksumAddress } from '@zilliqa-js/crypto';
 import namehash from './zns/namehash';
 import _ from 'lodash';
-import {ResolutionResult} from './types'
+import { ResolutionResult, SourceDefinition } from './types';
 
 const DefaultSource = 'https://api.zilliqa.com/';
-const registryAddress = 'zil1jcgu2wlx6xejqk9jw3aaankw6lsjzeunx2j0jz';
 const NullAddress = '0x0000000000000000000000000000000000000000';
 
+const NetworkIdMap = {
+  1: 'mainnet',
+  3: 'ropsten',
+  4: 'kovan',
+  42: 'rinkeby',
+  5: 'goerli',
+};
```

Comment:
For zilliqa that should be:

```
mainnet: 1
testnet: 333
```

Based on: https://apidocs.zilliqa.com/#blockchain-related-methods

Reply:
`
 networkIdMap: {
      1: 'zilliqa',
      333: 'dev',
      0: 'local',
    },
`

I used 0 as a network id for local testnet